rakelib/serialization.rake

```diff
@@ -8,12 +8,13 @@ task "test:serialize_fixtures" do
   $:.unshift(File.expand_path("../lib", __dir__))
```

Comment:
This task can be simplified further by being merged into the one below, since we have the FFI backend now and can serialize from JRuby (instead of having to use CRuby for that).

Reply:
Done